data/scripts.js

```diff
@@ -1174,6 +1174,23 @@ exports.BattleScripts = {
 		}
 
 		let movePool = (template.randomBattleMoves ? template.randomBattleMoves.slice() : Object.keys(template.learnset));
+
+		// Check for alternate move pool.
+		if (template[alternateMovePool]) {
+			let newMovePool = template[alternateMovePool].slice();
+
+			// Confirm it is a valid list of moves.
```

Comment:
...??? Why?

Do this in the tests if you have to; revalidating every time a battle starts is entirely the wrong decision...